packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -3255,6 +3255,188 @@ describe("IModelTransformerHub", () => {
     masterSeedDb.close();
   });
 
+  it("should fail older iModels without new versioning behavior unless ignoreNoBranchRelationshipData is true", async () => {
+    let targetScopeProvenanceProps: ExternalSourceAspectProps | undefined;
+    const setIgnoreNoBranchRelationshipData = (
+      transformer: IModelTransformer
+    ) => (transformer["_options"]["ignoreNoBranchRelationshipData"] = true);
+    const timeline: Timeline = [
+      { master: { 1: 1 } },
+      { master: { 2: 2 } },
+      { master: { 3: 1 } },
```

Comment:
is it important that these are separate changesets? why not:
```suggestion
      { master: { 1: 1, 2: 2, 3: 1 } },
```

Reply:
Just copied from other tests, changed it to 1 changeset.